responsive-grid-layout/onder/styles.css

```diff
@@ -0,0 +1,55 @@
+*{
+  margin: 0;
+  padding: 0; 
+  border: 1px blue solid;
+}
+body{
+  
+   margin: 0 ;
+   font-size: 26px;
+
+}
+
+.container {
+   height: 100vh;
+   display: grid;
+   grid-template-columns: 1fr 4fr 1fr;
+   grid-template-rows: 80px 1fr 80px;
+   grid-template-areas:  
+   "header header header"
+   "sidebar-left content sidebar-right"
+   "sidebar-left footer footer"
+}
+
+header {
+   display: grid;
```

Comment:
For the sake of this exercise, I think you only need to set `display: grid;` for the `div` with class 'container'. All other elements don't really have contain that you'd need to align.

Reply:
Tugba and I actually were doing the project together but both of us was wrong. good to know. We will correct all of that for the next assignment.